munge/srseng/toimportjson.ts

```diff
@@ -0,0 +1,99 @@
+import { execIfMain } from "execifmain";
+import { writeFile } from "fs/promises";
+import { Command } from "commander";
+import { join as joinPath, resolve as resolvePath } from "path";
+import { loadTsvFile } from "./util";
+import { Dictionary } from "./dictionary";
+import { Transducer } from "hfstol";
+
+const RESOURCE_DIR = resolvePath(
+  __dirname,
+  "..",
+  "..",
+  "src",
+  "srseng",
+  "resources"
+);
+
+const DICTIONARY_DIR = joinPath(RESOURCE_DIR, "dictionary");
+const FST_DIR = joinPath(RESOURCE_DIR, "fst");
+
+const relaxedAnalyzer = new Transducer(
+  joinPath(FST_DIR, "analyser-gt-desc.hfstol")
+);
+
+// TODO: the FST should recognize these forms directly, even if only in relaxed
+// mode.
+function headwordToFstLemma(headword: string) {
+  return headword
+    .replace(/[ā]/g, "a")
+    .replace(/[ī]/g, "i")
+    .replace(/[ū]/g, "u")
+    .replace(/[ō]/g, "o");
```

Comment:
Just curious: Can you explain why you bracket these characters in the regex even though they're only 1 character? Is that just a preferred convention?

Reply:
No, that’s just some leftover thing; I either originally started with more source characters in the square brackets, and didn’t remove the brackets when only a single character was left.